### DISCUSSION TITLE
I decided to share my (bad) experience with Penpot Desktop.

### DISCUSSION THREAD

**Discussion by datalot-369** [uxbox/uxbox#3584] [General]
I decided to share my (bad) experience with Penpot Desktop.
Don't get me wrong, I just want to describe why it is so bad,
And what I did expect in the first place. 

Me: You just realized penpot exists.
Me: Let's try it.
X: Penpot is not intended to be installable on a system.
X: However, if you want to do it, you need to:
X: Install penpot desktop (Page I).
Me: The official website is not clear on usage.
Me: After searching you find FAQ (Page II):
X: "You need to run your own instance to be able to use penpot desktop"
Me: Why is the purpose of Penpot Desktop btw?
Me: Anyways, let's install it...
X: You go to Penpot - Self-Host (Page III)
Me: Let's use docker (Page IV)
X: The tutorial says you need to get some docker configuration
X: You follow the link to install it (Page V)
Me: I'm using Linux, so let's follow the link Page VI)
Me: I'm using a Debian-based distribution, let's follow the link (Page VII)
X: You need to set up the Docker repository for your distribution, let's follow the link (Page VIII)

NOTE 1: At this point, you don't even have downloaded a thing to get penpot working offline, you are just browsing and browsing and browsing...
NOTE 2: My internet connection is pretty unstable, so at least a week has passed since I found penpot project and I started to find ways to download it.

Me: OK let's suppose I have downloaded Penpot Desktop, Docker, and Docker Desktop...
X: You also need the V2 Plugin to be able to go the the next page where we explain how to install penpot

Well... I give up.
Too many pages, too many steps, too many files, too many dependencies...
At this point, I prefer to use Figma or Penpot web app.
And yes, I could have installed penpot desktop at the end,
But it means I need to keep track of a bunch of packages that I will use for nothing more than penpot, and in the long term, migration or upgrading will be a total chaos.
In addition, I don't think that a containerized electron app could run softly on my (or any normal) system.

I would suggest integrating the whole process in a single bashscript or the penpot-desktop.deb itself (which build is already separated by distribution). I don't mind if the .deb weighs 1.5GB, my concern is to have 20 .deb packages from two or three additional repositories to run just one app.

I will use penpot anyways... and I hope I like it.

**Comment by Eptagone** [comment]
I agree. It could be really useful if the Penpot desktop app could work offline and without a backend server. To edit and save our designs on the computer for example.

**Reply by arunavo4** [reply]
Well maybe just a PWA will do it.
